addons/api/fnc_basicMissionSetup.sqf

```diff
@@ -82,14 +82,15 @@ private _addRadios = {
     _player setVariable [QGVAR(basicMissionSetup), true, true];
 
     private _cleanRadioList = [];
-    if !("ACRE_PRC343" in _defaultRadios) then {
+    private _defaultRadio = QEGVAR(sys_radio,defaultItemRadioType) call CBA_settings_fnc_get;
```

Comment:
That would be just `EGVAR(sys_radio,defaultItemRadioType)`, you don't need to call `get`.